alibi_detect/saving/tests/test_saving.py

```diff
@@ -204,7 +204,7 @@ def nlp_embedding_and_tokenizer(model_name, max_len, uae, backend):
 
     # Load tokenizer
     try:
-        tokenizer = AutoTokenizer.from_pretrained(model_name + 'TODO')
+        tokenizer = AutoTokenizer.from_pretrained(model_name)
```

Comment:
This had been accidentally introduced in a previous PR. It meant any tests using `nlp_embedding_and_tokenizer` were being skipped 😬